pkg/core/native/nativehashes/hashes.go

```diff
@@ -1,34 +1,69 @@
+// Code generated by "go generate go run gen.go"; DO NOT EDIT.
```

Comment:
Let's push this patch into another PR.

Reply:
Ref. https://github.com/nspcc-dev/neo-go/pull/3431.